src/main/java/bookstore/validation/FieldMatchValidator.java

```diff
@@ -0,0 +1,18 @@
+package bookstore.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.apache.commons.beanutils.BeanUtils;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch, Object> {
+    @Override
+    public boolean isValid(Object value, ConstraintValidatorContext context) {
+        try {
```

Comment:
example of implementation

```suggestion
        Object field = new BeanWrapperImpl(value).getPropertyValue(this.field);
        Object fieldMatch = new BeanWrapperImpl(value).getPropertyValue(this.fieldMatch);
        return Objects.equals(field, fieldMatch);
```